.github/workflows/test.yml

```diff
@@ -84,12 +90,49 @@ jobs:
         if: github.event_name != 'schedule'
         run: shellcheck -o all -S style -s sh $(find . -iname "*.sh")
 
-  # This job reports the results of the test matrix above
+  doc-checks:
+    name: Test documentation website
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Set up Node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: website/package-lock.json
+      - name: Install local website package
+        working-directory: ./website
+        run: npm ci
+      - name: Ensure website builds
+        working-directory: ./website
+        run: npm run build
+
+  # This job reports the results of the test jobs above and is used to enforce status checks in
+  # the repo settings without needing to update those settings everytime the test jobs are updated.
   test:
     if: always()
-    needs: test-matrix
     runs-on: ubuntu-latest
+    needs: [test-matrix, deno-checks, shellcheck, doc-checks]
     steps:
-      - if: needs.test-matrix.result != 'success'
-        name: Fail the build
-        run: exit 1
+      - name: Check for test jobs failure
+        if: >
+          (needs.test-matrix.result != 'success')
+          || (needs.deno-checks.result != 'success')
+          || (needs.shellcheck.result != 'success')
+          || (needs.doc-checks.result != 'success')
+        run: |
+          echo "At least one test job was not successful"
+          exit 1
+
+      - name: Confirm test jobs success
+        if: >
+          (needs.test-matrix.result == 'success')
+          && (needs.deno-checks.result == 'success')
+          && (needs.shellcheck.result == 'success')
+          && (needs.doc-checks.result == 'success')
+        run: echo "All test jobs were successful"
```

Comment:
Doing the same thing twice just to print a different message is pretty pointless imo. Developers should be able to distinguish a failed build from a successful one.

Reply:
That is a good point. This will be changed.